### DISCUSSION TITLE
Walk-through video on the README.md

### DISCUSSION THREAD

**Discussion by adriangalilea** [webdev23/shelix#6] [Ideas]
Super interesting project,

I'd say the introduction part of it can improve by having a walk-through video that covers, in this order:
- what is the usage flow
- how to set it up

Short and to the point. Would make easier to understand what the project is about.

P.S.: I would make it clearer if it provides a file-tree experience to Helix, which I think it does, as I feel the other features are nice to haves but the file-picker is what people are missing on Helix.

**Comment by webdev23** [comment]
Thanks for the heads-up, a walk-through video and more documentation is certainly on the top priority.
